README.md

```diff
@@ -54,14 +56,6 @@ What it's bad for:
 * Darwin configurations
 * devshell
 
-## Blacklisted inputs
-
-In order to avoid name clashes, avoid loading inputs with the following names:
-* lib
-* pname
-* system
-* pkgs
-
 ## Packages folder
 
 If the ./pkgs folder exists, load every sub-folder in it and map it to the `packages` output.
```

Comment:
Did this not recently become `packages` folder?

Reply:
Thanks, removed this old section